### ISSUE TITLE
UnexpectedValueException when checking smw_hash field consistency

### ISSUE THREAD

**Issue by coyotefather** [SemanticMediaWiki/SemanticMediaWiki#5574]
I was upgrading from MW version 1.40.2 and encountered the below error when running upgrade.php. The same error is returned when running setupStore.php. I wasn't sure if this version of MW was supported yet or not by SMW. From what I could tell, MW 1.40.2 and SMW 4.1.2 were working fine together.

MediaWiki version: 1.41.0
SMW version: 4.1.2

```
Checking smw_hash field consistency ...
UnexpectedValueException from line 266 of /home/****/public_html/wiki/extensions/SemanticMediaWiki/src/MediaWiki/Connection/Database.php:Expected a ResultWrapper for
smw_object_ids
Array
Array
#0 /home/****/public_html/wiki/extensions/SemanticMediaWiki/maintenance/populateHashField.php(216): SMW\MediaWiki\Connection\Database->select('smw_object_ids', Array, Array, 'SMW\\Maintenance...', Array)
#1 /home/****/public_html/wiki/extensions/SemanticMediaWiki/src/SQLStore/TableBuilder/Examiner/HashField.php(69): SMW\Maintenance\populateHashField->fetchRows()
#2 /home/****/public_html/wiki/extensions/SemanticMediaWiki/src/SQLStore/TableBuilder/TableBuildExaminer.php(145): SMW\SQLStore\TableBuilder\Examiner\HashField->check()
#3 /home/****/public_html/wiki/extensions/SemanticMediaWiki/src/SQLStore/Installer.php(239): SMW\SQLStore\TableBuilder\TableBuildExaminer->checkOnPostCreation(Object(SMW\SQLStore\TableBuilder\MySQLTableBuilder))
#4 /home/****/public_html/wiki/extensions/SemanticMediaWiki/src/SQLStore/SQLStore.php(456): SMW\SQLStore\Installer->install(Object(SMW\Options))
#5 /home/****/public_html/wiki/extensions/SemanticMediaWiki/src/Store.php(490): SMW\SQLStore\SQLStore->setup(Object(SMW\Options))
#6 /home/****/public_html/wiki/includes/installer/DatabaseUpdater.php(556): SMW\Store::setupStore(true, Object(SMW\Options))
#7 /home/****/public_html/wiki/includes/installer/DatabaseUpdater.php(524): DatabaseUpdater->runUpdates(Array, true)
#8 /home/****/public_html/wiki/maintenance/update.php(185): DatabaseUpdater->doUpdates(Array)
#9 /home/****/public_html/wiki/maintenance/includes/MaintenanceRunner.php(703): UpdateMediaWiki->execute()
#10 /home/****/public_html/wiki/maintenance/run.php(51): MediaWiki\Maintenance\MaintenanceRunner->run()
#11 {main}
```

**Comment by Nikerabbit** [comment]
This is fixed in SMW master. Maybe worth a new release at some point?

**Comment by coyotefather** [comment]
I normally only use composer to update but I can download and replace with master until there's a new release.

**Comment by Nikerabbit** [comment]
Fix included in 4.1.3.
